services/mozilla-observatory/mozilla-observatory.service.js

```diff
@@ -0,0 +1,133 @@
+'use strict'
+
+const { BaseJsonService } = require('..')
+
+const Joi = require('joi')
+const schema = Joi.object({
+  state: Joi.string()
+    .valid('ABORTED', 'FAILED', 'FINISHED', 'PENDING', 'STARTING', 'RUNNING')
+    .required(),
+  grade: Joi.alternatives()
+    .when('state', {
+      is: 'FINISHED',
+      then: Joi.string().regex(/^[ABCDEF][+-]?$/),
+      otherwise: Joi.only(null),
+    })
+    .required(),
+  score: Joi.alternatives()
+    .when('state', {
+      is: 'FINISHED',
+      then: Joi.number()
+        .integer()
+        .min(0)
+        .max(200),
+      otherwise: Joi.only(null),
+    })
+    .required(),
+}).required()
+
+const documentation = `
+<p>
+  The <a href="https://observatory.mozilla.org">Mozilla HTTP Observatory</a>
+  is a set of tools to analyze your website
+  and inform you if you are utilizing the many available methods to secure it.
+</p>
+</p>
+  By default the scan result is hidden from the public result list.
+  You can activate the publication of the scan result
+  by setting <code>publish</code> parameter to <code>true</code>
+<p>
+<p>
+  The badge returns a cached site result if the site has been scanned anytime in the previous 24 hours.
+  If you need to force invalidating the cache,
+  you can to do it manually through the <a href="https://observatory.mozilla.org">Mozilla Observatory Website</a>
+</p>
+`
+
+module.exports = class MozillaObservatory extends BaseJsonService {
+  static get category() {
+    // TODO: Once created, change to a more appropriate category,
+    // see https://github.com/badges/shields/pull/2926#issuecomment-460777017
+    return 'monitoring'
+  }
+
+  static get route() {
+    return {
+      base: 'mozilla-observatory',
+      pattern: ':which(grade|grade-score)/:host',
+      queryParams: ['publish'],
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Mozilla HTTP Observatory Grade',
+        namedParams: { which: 'grade', host: 'github.com' },
```

Comment:
In order to make the examples render properly, they'll need to include a `pattern` which specifies the value for `which`. 

```js
pattern: 'mozilla-observatory/grade/:host',
namedParams: { host: 'github.com' },
```